lib/minitest_ext/exercism_reporter.rb

```diff
@@ -30,16 +25,16 @@ def report
     end
 
     def status
-      tests.all?{|t|t[:status] == :pass} ? :pass : :fail
+      tests.all? { |t| t[:status] == :pass } ? :pass : :fail
     end
 
-    def exception_raised!(e)
+    def exception_raised!(exc)
```

Comment:
Is this Rubocop too?

Reply:
It is. This was the rule about an identifier having to be at least three characters long. Is that okay, or do you want me to ignore it?

Reply:
Yeah - ignore that rule please. 

We should somewhere keep a global rubocop file with all these in that can act as our base, as we're discovering extra useful things in every repo but they're all getting out of sync.

Reply:
Yes, please do use the common names for the pretty much industry wide standards.  `i` for index rather than `ndx` or something else.  `e` for `exception` as it is a pronoun that is clearly defined by proximity, etc.